src/widgets/copilot/components/Messages/LocationModal.tsx

```
@@ -0,0 +1,522 @@
+              // Only set zoom initially, not on every update
+              locationInitialized.current
+                ? leafletMapRef.current.getZoom()
+                : 15,
+              { animate: false },
+            );
+          }
+        }, 300);
+
+        locationInitialized.current = true;
+      },
+      (_) => {
```

ESLint

[error] '_' is defined but never used.

(@typescript-eslint/no-unused-vars)